swagger.yaml

```diff
@@ -2257,6 +2257,16 @@ paths:
             $ref: '#/definitions/error'
 
 definitions:
+  operators:
+    type: object
+    properties:
+      ocs:
```

Comment:
OCS is a particular type of operator, not a property of operators. Could this be defined better?

Is it possible to define an interface for operators with a Validate function? Then the assisted-service would just iterate over the array of operators and call the Validate function for each.

Reply:
I see this as a tighter integration than just enabling validations.  We would like to accept parameters, validate them against the discovered hardware, and create the yamls necessary for configuring the system as requested.  So OCS is a type of operator that we recognize, and in the future can accept more parameters.  Likewise, in the future we will likely support other operators similarly and want to group them under "operators".

Reply:
That sounds good but defining OCS as a property doesn't seem to fit.
Is there a different way to define ocs as a type of operator instead of a property of operators?

Reply:
I'm not sure what you mean.